assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/positive.yaml

```diff
@@ -0,0 +1,44 @@
+#this is a problematic code where the query should report a result(s)
+Resources:
+  EMRSecurityConfiguration:
+    Type: AWS::EMR::SecurityConfiguration
+    Properties:
+       Name: String
+       SecurityConfiguration:
+         EncryptionConfiguration:
+           EnableInTransitEncryption: false
+           EnableAtRestEncryption: false
+           AtRestEncryptionConfiguration:
+             LocalDiskEncryptionConfiguration:
+                 EnableEbsEncryption: true
+                 EncryptionKeyProviderType: AwsKms
+                 AwsKmsKey: arn:aws:kms:us-east-1:123456789012:key/12345678-1234-1234-1234-123456789012
+---
+Resources:
+  EMRSecurityConfiguration-01:
+    Type: AWS::EMR::SecurityConfiguration
+    Properties:
+       Name: String
+       SecurityConfiguration:
+         EncryptionConfiguration:
+           AtRestEncryptionConfiguration:
+             LocalDiskEncryptionConfiguration:
+                 EnableEbsEncryption: false
+---
+Resources:
+  EMRSecurityConfiguration-03:
```

Comment:
Remove `-` :
`E3006 Resource EMRSecurityConfiguration-03 has invalid name.  Name has to be alphanumeric.`